### PR TITLE
Check form Symmetric NAT in stack checks

### PR DESCRIPTION
**Pull request by manuquentin**
See https://webrtchacks.com/symmetric-nat/

### CODE REVIEW

File: src/domain/Contact.js [hunk 1]
```diff
@@ -276,25 +276,31 @@ export default class Contact {
...
+          backend: edge.node.wazoBackend,
+          source: edge.node.wazoSourceName,
+          sourceId: edge.node.wazoSourceEntryId,
+          email: email || '',
+          emails: email ? [{ label: 'primary', email }] : [],
+          uuid: edge.node.userUuid,
+        });
+      }).filter(contact => !!contact);
+    };
   }
```
Comment (bogue) [inline src/domain/Contact.js:304]: Are you ok with this being in here? feel a little off-topic

File: src/checker/checks/ice-ipv4.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    }
+
+    const pc = new RTCPeerConnection({
+      iceServers: [
+        { urls: 'stun:stun1.l.google.com:19302' },
+        { urls: 'stun:stun2.l.google.com:19302' },
+      ],
+    });
+
+    pc.createDataChannel('wazo-check-nat');
```
Comment (bogue) [inline src/checker/checks/ice-ipv4.js:28]: Are we using this? If so, is the label right?

File: src/checker/checks/symmetric-nat.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    }
+
+    const pc = new RTCPeerConnection({
+      iceServers: [
+        { urls: 'stun:stun1.l.google.com:19302' },
+        { urls: 'stun:stun2.l.google.com:19302' },
+      ],
+    });
+
+    pc.createDataChannel('wazo-check-nat');
```
Comment (bogue) [inline src/checker/checks/symmetric-nat.js:19]: Are we using this?

**Review by bogue — APPROVED** [review]

File: src/checker/checks/symmetric-nat.js (hunk 3 above)
Comment (manuquentin) [reply src/checker/checks/symmetric-nat.js:19]: Yes

**Pull request merged**